src/coreclr/jit/codegenwasm.cpp

```diff
@@ -164,6 +178,123 @@ void CodeGen::genCodeForBinary(GenTreeOp* treeNode)
         case PackOperAndType(GT_ADD, TYP_DOUBLE):
             ins = INS_f64_add;
             break;
+
+        case PackOperAndType(GT_SUB, TYP_INT):
```

Comment:
Now that we're filling more of this out, there should be an NYI for `gtOverflow()` (checked operations).